homework/index.js

```diff
@@ -16,31 +30,97 @@
     xhr.send();
   }
 
-  function createAndAppend(name, parent, options = {}) {
-    const elem = document.createElement(name);
-    parent.appendChild(elem);
-    Object.keys(options).forEach(key => {
-      const value = options[key];
-      if (key === 'text') {
-        elem.textContent = value;
-      } else {
-        elem.setAttribute(key, value);
-      }
+  function renderRepositories(repo, ulRepoInfo) {
+    const url = `https://api.github.com/repos/HackYourFuture/${repo}`;
+    fetchJSON(url, (err, repoInfo) => {
+      ulRepoInfo.innerHTML = '';
+      const li = createAndAppend('li', ulRepoInfo, { id: 'li1' });
```

Comment:
An `ul` with a single `li` child containing a single `table` does not make sense. You do not need a list (of one element) here. This is the HTML you generate:

```html
<div class="info-section-div">
  <div>
    <ul class="li-container">
      <li id="li1">
        <table>
          <tbody>
            <tr>
              <td class="label">Repository:</td>
              <td><a href="https://github.com/HackYourFuture/alumni" target="_blank">alumni</a></td>
            </tr>
            <tr>
              <td class="label">Description:</td>
              <td>Alumni Guides, everything you need to land a job</td>
            </tr>
            <tr>
              <td class="label">Forks:</td>
              <td>9</td>
            </tr>
            <tr>
              <td class="label">Updated:</td>
              <td>6/21/2019, 1:38:21 PM</td>
            </tr>
          </tbody>
        </table>
      </li>
    </ul>
  </div>
  <div>
    <ul class="li-container">
      <li class="li2">
        <img src="https://avatars0.githubusercontent.com/u/15912395?v=4" alt="gijscor photo" />
        <h1><a href="https://github.com/gijscor" target="_blank">gijscor</a></h1>
        <p>47</p>
      </li>
      <li class="li2">
        <img src="https://avatars1.githubusercontent.com/u/37488847?v=4" alt="wouterkleijn photo" />
        <h1><a href="https://github.com/wouterkleijn" target="_blank">wouterkleijn</a></h1>
        <p>11</p>
      </li>
    </ul>
  </div>
</div>
```